### PR TITLE
redmine: 3.4.6 -> 3.4.8

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
Version bump.

Fairly minor changelog from 3.4.6 to 3.4.8: https://www.redmine.org/projects/redmine/wiki/Changelog_3_4

Note: I have tested this on 18.09 as well so this should be back ported.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by aanderse** [comment]
@GrahamcOfBorg test redmine

File: pkgs/applications/version-management/redmine/Gemfile.lock [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.2)
+    addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
     arel (6.0.4)
     builder (3.2.3)
-    capybara (3.9.0)
+    capybara (2.18.0)
```
[the remark this answers is not in the document]
Comment (aanderse) [reply pkgs/applications/version-management/redmine/Gemfile.lock]: Dependency changed in the Gemfile between the two versions. Not relevant though, as capybara is declared in the "test" group which isn't used. So my understanding of ruby + nix is that while this file will be downloaded, it isn't actually used and therefore unimportant.

[the remark this answers is not in the document]
Comment (Ekleog) [reply pkgs/applications/version-management/redmine/Gemfile.lock]: Sounds right, thank you!

**Comment by Ekleog** [comment]
@garbas cc as a maintainer, I'm merging as it looks right, feel free to revert if I missed something

**Comment by Ekleog** [comment]
BTW, @aanderse , you could consider adding yourself as a maintainer, as you appear to have done all the recent work on it :)

**Comment by aanderse** [comment]
@Ekleog Thank you. Yeah I guess at this point that makes sense.

Please backport this to 18.09 as well as it does have some (minory) security fixes!

**Comment by Ekleog** [comment]
@aanderse I've been trying to run `nix-build nixos/tests/redmine.nix` locally and it fails (both before and after the change on both release-18.09 and master). Could you open a PR for the backport (done with `git cherry-pick -x`) so that ofborg could run against it?

**Pull request merged**